win64.mak

```diff
@@ -481,6 +481,17 @@ cov : $(SRC_TO_COMPILE) $(LIB)
 
 html : $(DOCS)
 
+################### Win32 COFF support #########################
+
+# default to 32-bit compiler relative to 64-bit compiler, link and lib are architecture agnostic
```

Comment:
The VC 32-bit compiler is in the parent folder of the folder that contains the 64-bit compiler, e.g. for the default CC="$(VCDIR)\bin\amd64\cl", the 32-bit compiler is CC="$(VCDIR)\bin\cl". Maybe there is a better wording...